.cursor/.rules/cursorules.mdc

```diff
@@ -20,7 +21,7 @@ Code Style and Structure
 - Use descriptive variable names with auxiliary verbs (e.g., isLoading, hasError).
 - Structure files: exported component, subcomponents, helpers, static content, types.
 - When creating a component, use arrow functions
-- Always opt to use a styled component over a plain Text or View component
+- Prefer styled-components and MUI components over inline styles and unstyled HTML elements
```

Comment:
I know we're not currently planning a mobile variant for Yurt, but I wonder if down the road it'd be worth at least making variants of these rules to account for some different libraries we use. For example, I'm not sure how problematic this rule with `alwaysApply: true` would be on a mobile app using Tamagui instead of MUI. 

It's probably easier to just modify these on a project-by-project basis, but figured I'd flag